pkg/opentelemetry/controller.go

```diff
@@ -38,6 +40,16 @@ const (
 	otelServiceNameEnvVar = "OTEL_SERVICE_NAME"
 )
 
+var (
+	// releaseVersion = ??? // TODO: reference version of the build to include in outward comms
+	// start of this autoinstrumentation's exporter User-Agent header, e.g. ""OTel-Go-Auto-Instrumentation/1.2.3"
+	baseUserAgent = fmt.Sprintf("OTel-Go-Auto-Instrumentation") // TODO: include the releaseVersion
+	// Information about the runtime environment for inclusion in User-Agent
+	runtimeInfo = fmt.Sprintf("%s (%s/%s)", strings.Replace(runtime.Version(), "go", "go/", 1), runtime.GOOS, runtime.GOARCH)
+	// The default User-Agent when no additions have been given
+	autoinstUserAgent = fmt.Sprintf("%s %s", baseUserAgent, runtimeInfo)
```

Comment:
An example of what the user-agent output looks like from an Ubuntu AMD64 host running this branch at this commit:

```
OTel-Go-Auto-Instrumentation go/1.18.1 (linux/amd64) grpc-go/1.54.0
```

The `baseUserAgent` string was an initial stab at an ID string. Totally happy with a better identifier if someone's got one.

Reply:
With the addition of `releaseVersion`, the user agent now looks like:

```
OTel-Go-Auto-Instrumentation/v0.1.0-alpha go/1.18.1 (linux/amd64) grpc-go/1.54.0
```